layouts/partials/fragments/search.html

```diff
@@ -0,0 +1,76 @@
+{{- $bg := .Params.background | default "light" -}}
+{{- $self := . -}}
+{{- .page_scratch.Add "js" ("syna-search.js" | relLangURL) }}
+
+{{ "<!-- Pricing -->" | safeHTML }}
+<section id="{{ .Name }}">
+  <div class="overlay container-fluid
+    {{- printf " bg-%s" $bg -}}
+  ">
+  <div class="container py-5">
+    {{- with .Params.title }}
+      <div class="col text-center
+        {{- if or (eq $bg "white") (eq $bg "light") (eq $bg "secondary") (eq $bg "primary") -}}
+          {{- printf " text-%s" "dark" -}}
+        {{- else -}}
+          {{- printf " text-%s" "light" -}}
+        {{- end -}}
+        {{- if and (not (isset $self.Params "subtitle")) (not $self.Content) -}}
+          {{- printf " pb-4" -}}
+        {{- end -}}
+      ">
+        <h2>
+          {{- . | markdownify -}}
+        </h2>
+      </div>
+    {{- end -}}
+    {{- with .Params.subtitle -}}
+      <div class="row">
+        <div class="col text-center pt-2 pb-4
+          {{- if or (eq $bg "secondary") (eq $bg "primary") -}}
+            {{- printf " text-%s" "dark" -}}
+          {{- else -}}
+            {{- printf " text-muted text-%s" "secondary" -}}
+          {{- end -}}
+        ">
+          <h6>
+            {{- . | markdownify -}}
+          </h6>
+        </div>
+      </div>
+    {{- end }}
+    <div class="row">
+      <form action="{{ "search" | absURL }}" class="w-100">
+        <div class="col-md-6 col-12">
+          <div class="form-group row">
+            <label for="search-query" class="col-2 pl-0 pt-2">Search:</label>
+            <input type="text" class="form-control col-10 col-md-8" id="search-query-{{ .Name }}" name="s" placeholder="Search for something...">
+          </div>
+        </div>
+      </form>
+    </div>
+    <div class="row">
+      <div class="col px-0">
+        <h3>Matching pages</h3>
+        <div id="search-results-{{ .Name }}" class="col px-0">
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+<!-- this template is sucked in by search.js and appended to the search-results div above. So editing here will adjust style -->
+<script id="search-result-template-{{ .Name }}" type="text/x-js-template">
```

Comment:
Maybe use a helper partial for the template so it's consistent for navbar and search fragment?